### PR TITLE
doc: add PowerShell install instructions added

### LINKED ISSUE #9467: Document the use of Add-AppxPackage
Reading README.md I could not find how to install Windows Terminal from the command line when the Microsoft Store is not available, however I did find https://github.com/microsoft/terminal/pull/6942 which worked perfectly but looks like it was never merged. I want to raise a PR to update README.md because in Windows Sandbox the Microsoft Store is not available, and this PowerShell solution is the quickest and easiest way to get Windows Terminal installed in the sandbox, as you can't just double-click the .msixbundle file.

I am creating this issue before raising a PR, as I believe this to be the correct process.

### PR DESCRIPTION
**Pull request by glawrence**
Updated README.md to add instructions for installing via the command
line. This is useful when the Microsoft Store is not available which is
the case in Windows Sandbox.

This completes what was started in #6942
Closes #9467

### CODE REVIEW

**Review by zadjii-msft — APPROVED** [review]
Yea that's good enough for me. Thanks!

File: README.md [hunk 1]
```diff
@@ -33,10 +33,20 @@ This is our preferred method.
...
-builds can be manually downloaded from this repository's [Releases
+For users who are unable to install Windows Terminal from the Microsoft Store,
+released builds can be manually downloaded from this repository's [Releases
 page](https://github.com/microsoft/terminal/releases).
 
+Download the `Microsoft.WindowsTerminal_<versionNumber>.msixbundle` file from
+the **Assets** section. To install the app, you can simply double-click on the
+`.msixbundle` file, and the app installer should automatically run. If that
+fails for any reason, you can try the following command at a PowerShell prompt
+instead:
```
Comment (DHowett) [inline README.md:44]:
```suggestion
fails for any reason, you can use the following command at a PowerShell prompt:
```

File: README.md [hunk 2]
```diff
@@ -33,10 +33,20 @@ This is our preferred method.
...
+
+```powershell
+Add-AppxPackage Microsoft.WindowsTerminal_<versionNumber>.msixbundle
```
Comment (DHowett) [inline README.md:47]:
```suggestion
# NOTE: If you are using PowerShell 7+, please run
# Import-Module Appx -UseWindowsPowerShell
# before using Add-AppxPackage.

Add-AppxPackage Microsoft.WindowsTerminal_<versionNumber>.msixbundle
```

Comment (DHowett) [reply README.md:47]: sound okay? should this be promoted to a toplevel note?

**Review by DHowett — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (glawrence) [reply README.md:47]: Yes, that sounds great, thanks for the build, I never considered anyone jumping straight to PowerShell 7+

**Pull request merged**